index.html

```diff
@@ -0,0 +1,48 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Alexander Beaumier - Personal Website</title>
+    <link rel="stylesheet" href="css/Styles.css"> 
+    <style>
```

Comment:
while this definitely works, consider avoiding in-line style tags in an HTML page because it makes this less reusable. Also, I'm unsure if the @ keyframes will work here because it's in an HTML page instead of a .css file